### ISSUE TITLE
Formatting twice changes result of mathematical expression

### ISSUE THREAD

**Issue by sgruetter** [prettier/plugin-php#1421]
**@prettier/plugin-php v0.14.0**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCACsAOj4gWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDaRTtrMMEEJvBAAHgB+YzAAHUwtgHodgAJRaih9qRLx-YQfE6l+-zhuy+P96LY47H22aH3oH3QAN0eAAozpgoABKOxbAAkmEEPg4SH2AF59jYANww6DYQSyGAo-ZA7L7ABUJP2sPhiIg4P2mX2AEZMdsWXt9qRjphLmwANYqJ4nbG4-G3CA+AWnc4iu5sOD8tiYCj7MBjBCYZT7XpMXYHMqwBBItgUCgfV7vTVcmgnADuECwGrAsjehn2MWoCuU-KtPygOLxhI4RshMLhCKRqIxWN9woJxLJFND1LpjPRIHaQA)

**Input:**
```php
<?php

// The first time formatted the braces are removed (fine).
$number = 5;
$result = (2 ** $number) - 1;


// When taking the result form the first formatting and pasting it,
// Prettier adds braces in the wrong place, changing the result (bad).
$number = 5;
$result = 2 ** $number - 1;
```

**Output:**
```php
<?php

// The first time formatted the braces are removed (fine).
$number = 5;
$result = 2 ** $number - 1;

// When taking the result form the first formatting and pasting it,
// Prettier adds braces in the wrong place, changing the result (bad).
$number = 5;
$result = 2 ** ($number - 1);

```

As written in the comments above, Prettier changes the result when formatting an initial piece of code twice or more times.

Adding `// prettier-ignore` a line before works temporarily if (!) the change is noticed.

As this issue might have very serious implications, a quick fix would be appreciated highly.

If you need some more information or if I can be of assistance, please let me know.

**Comment by czosel** [comment]
Thanks for the bug report!

**Comment by sgruetter** [comment]
No problem, thanks for Prettier for PHP :-)

**Comment by czosel** [comment]
I took a closer look - this turns out to be an issue with operator precedence in the parser.

**Comment by czosel** [comment]
We just released `v0.14.2` which fixes this :tada:

**Comment by sgruetter** [comment]
Awesome, it works like a charm. Thanks a lot @czosel!
